src/commonTest/kotlin/komem.litmus/infra/TestDefaults.kt

```diff
@@ -0,0 +1,21 @@
+package komem.litmus.infra
+
+import komem.litmus.LTDefinition
+import komem.litmus.LTOutcomeType
+import komem.litmus.LTRunParams
+import komem.litmus.LTRunner
+import kotlin.test.assertTrue
+
+expect val defaultParams: LTRunParams
+expect val defaultRunner: LTRunner
+
+fun LTDefinition<*>.run(
+    params: LTRunParams = defaultParams,
+    runner: LTRunner = defaultRunner,
+) {
+    val results = runner.runTest(params, this)
+    assertTrue { results.none { it.type == LTOutcomeType.FORBIDDEN } }
```

Comment:
why isn't it a Kotlin/JUnit test?

Reply:
This is a wrapper for writing actual tests, and it uses `kotlin.test.assertTrue`. What do you mean?

Reply:
Sorry, I thought this is a test on its own.